lib/std/crypto/benchmark.zig

```diff
@@ -458,7 +460,9 @@ fn mode(comptime x: comptime_int) comptime_int {
 }
 
 pub fn main() !void {
-    const stdout = std.fs.File.stdout().deprecatedWriter();
+    var stdout_buffer: [4096]u8 = undefined;
```

Comment:
If you flush after every write, there's no need for a buffer. Recommend to either pass an empty buffer, and remove all the flush() calls, or flush only once at the end.